src/view/boot.ts

```diff
@@ -18,6 +18,7 @@ import {MdIconRegistry} from '@angular2-material/icon/icon';
 import {CoreWebService} from "../api/services/core-web-service";
 import {LoginService} from "../api/services/login-service";
 import {RoutingService} from "../api/services/routing-service";
+import {FormatDate} from "../api/services/format-date-service";
```

Comment:
I'm injecting this from the beginning because this service in in charge of setting the language for the date and we need to share it across the app.